global/prometheus-alertmanager-operated/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 type: application
 description: Prometheus Alertmanager via operator.
 name: prometheus-alertmanager-operated
-version: 4.5.2
+version: 4.6.0
```

```suggestion
version: 4.5.3
```

Here we can still go with patch :). From my original comment:

> Please bump the minor version of prometheus-vmware-rules and the patch version prometheus-alertmanager-operated.